all-tests.nix

```diff
@@ -100,8 +100,20 @@ in
           client.succeed("ob thunk pack ~/code/myapp")
           client.succeed("grep -qF 'git' ~/code/myapp/default.nix")
           client.succeed("grep -qF 'myorg' ~/code/myapp/git.json")
+          client.succeed("ob thunk unpack ~/code/myapp")
       
+      with subtest("test obelisk can set the public / private flag"):
+          client.succeed("ob thunk pack ~/code/myapp --private")
+          client.fail("""grep -qF '"private": practice' ~/code/myapp/git.json""")
+          client.succeed("""grep -qF '"private": true' ~/code/myapp/git.json""")
+          client.succeed("ob thunk unpack ~/code/myapp")
+          client.succeed("ob thunk pack ~/code/myapp --public")
+          client.succeed("""grep -qF '"private": false' ~/code/myapp/git.json""")
+          client.succeed("ob thunk unpack ~/code/myapp")
```

Comment:
We also need to test that doing something with nix on the thunk works. Like `nix eval "import  ~/code/myapp"` or something (check that this actually forces the nix to pull down the thunk).

Reply:
done in https://github.com/obsidiansystems/obelisk/pull/619/commits/2e0b7698b36c1301ac73113c7c2cda6460cea3e5

Reply:
I think you meant the `-v` change was done not this one?

Reply:
right

Reply:
something with the thunk done in  https://github.com/obsidiansystems/obelisk/pull/619/commits/1f3ee0ee6ce9d572a4a7de8e8658bd4e7a3af343